deployment/cfn/vpc.py

```diff
@@ -49,20 +46,17 @@ class VPC(StackNode):
 
     ATTRIBUTES = {'StackType': 'StackType'}
 
-    PUBLIC_SUBNETS = []
-    PRIVATE_SUBNETS = []
-
     _NAT_SECURITY_GROUP_CACHE = None
 
     def set_up_stack(self):
         super(VPC, self).set_up_stack()
 
-        tags = self.get_input('Tags').copy()
-        tags.update({'StackType': 'VPC'})
-
-        self.default_tags = tags
+        self.default_tags = self.get_input('Tags').copy()
         self.region = self.get_input('Region')
-        self.availability_zones = get_availability_zones(self.aws_profile)
+        self.availability_zones = get_availability_zones(self.aws_profile,
```

Comment:
Before applying these changes, the stack JSON would have random AZs and CIDR ranges every time. This allows us to pin to specific AZs and CIDR ranges, like we do in Terraform, so we can update the existing VPC stack (without replacing everything).